### ISSUE TITLE
>| is not seen often, warning could be more helpful, is understood only after reading up on the operator

### ISSUE THREAD

**Issue by smoe** [koalaman/shellcheck#3133]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2188
- My shellcheck version: 0.10.0 and online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
halmsg="somefile"
>|$halmsg
```

#### Here's what shellcheck currently says:
```
Line 3	SC2188: This redirection doesn't have a command. Move to its command (or use 'true' as no-op).
```
#### Here's what I wanted or expected to see:

Is "true >| $halmsg" what you wanted to write to very explicitly empty that file and override a potential noclobber?
